src/MOI_wrapper.jl

```diff
@@ -58,14 +60,32 @@ mutable struct Optimizer <: MOI.AbstractOptimizer
     maxsense::Bool
     data::Union{Nothing, ModelData} # only non-Nothing between MOI.copy_to and MOI.optimize!
     sol::Solution
-    options
+    silent::Bool
+    options::Dict{Symbol, Any}
     function Optimizer(; kwargs...)
-        new(ConeData(), false, nothing, Solution(), kwargs)
+        optimizer = new(ConeData(), false, nothing, Solution(), false, Dict{Symbol, Any}())
+        for (key, value) in kwargs
+            MOI.set(optimizer, MOI.RawParameter(key), value)
+        end
+        return optimizer
     end
 end
 
 MOI.get(::Optimizer, ::MOI.SolverName) = "ECOS"
 
+function MOI.set(optimizer::Optimizer, param::MOI.RawParameter, value)
+    optimizer.options[param.name] = value
+end
+function MOI.get(optimizer::Optimizer, param::MOI.RawParameter)
+    return optimizer.options[param.name]
```

Comment:
TODO: This gives a poor error message if the name of the parameter is invalid.

Reply:
We are currently lacking a convention for that. Even `UniversalFallback` do not have a nice error:
https://github.com/JuliaOpt/MathOptInterface.jl/blob/52d166f91614d5d522bf055cac232c7c5ba3ea96/src/Utilities/universalfallback.jl#L104-L105
Should it be a `AttributeNotSetError` ? Or just `nothing` ?